### DISCUSSION TITLE
Saleor SDK (WIP)

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#7765] [Announcements]
A new Saleor SDK is under development. We've decided to make the SDK more flexible and minimal to follow our GraphQL-first approach.

You can find more info on our [Twitter](https://twitter.com/getsaleor/status/1422895400239910913)

If you have any questions, feel free to let us know!

**Comment by crisfcodes** [comment]
@karolkielecki This new version is compatible with React Native?
